### DISCUSSION TITLE
Compile error with release 2.4.15.4 if libhiredis-dev is not installed

### DISCUSSION THREAD

**Discussion by HolgerHees** [OpenIDC/mod_auth_openidc#1194] [General]
after upgrading from version 2.4.15.3 to 2.4.15.4 I was not able to compile openidc again. I got the following error

`src/config.c: In function 'oidc_set_redis_connect_timeout':",
 src/config.c:1358:76: error: 'oidc_cfg' has no member named 'cache_redis_connect_timeout'",
  1358 |                 rv = oidc_parse_redis_connect_timeout(cmd->pool, arg1, &cfg->cache_redis_connect_timeout);",
 src/config.c:1360:70: error: 'oidc_cfg' has no member named 'cache_redis_keepalive'",
  1360 |                 rv = oidc_parse_redis_keepalive(cmd->pool, arg2, &cfg->cache_redis_keepalive);",
 make: *** [Makefile:942: src/config.lo] Error 1"`

my workaround was to install libhiredis-dev but I don't use redis in my openidc setup.

I guess it is related to this commit https://github.com/OpenIDC/mod_auth_openidc/commit/603a51be18290ef5eab2dce6096790b3b25957e8

The reason why I create an issues here is, because your issues tracker does not provide an option to create an issues. I hope this is the right place.

**Comment by zandbelt** [comment]
thanks for reporting, tracked and fixed in #1195

**Comment by HolgerHees** [comment]
confirmed, that it compiles now without this lib. Thanks!
